com/yadro/HWManager/StorageManager.interface.yaml

```diff
@@ -0,0 +1,62 @@
+description: >
+    The interface to control Yadro storage subsystem.
+
+methods:
+    - name: FindDrive
+      description: >
+          Return port name where the rive insrted.
+      parameters:
+          - name: DriveSN
+            type: string
+            description: >
+                Serial number of the drive, to looking for
+      returns:
+          - name: Result
+            type: string
```

Comment:
as I said, a tuple like `[string location, int slot]` could be useful, e.g. `['front', 7]`.
that would be much more i18n-friendly.